### PR TITLE
Update to otel-java 0.5.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by iNikem**

### CODE REVIEW

**Comment by iNikem** [comment]
See #463

File: agent-tooling/src/main/java/io/opentelemetry/auto/tooling/TracerInstaller.java [hunk 1]
```diff
@@ -70,10 +70,12 @@ private static synchronized void installExportersFromJar(final String exporterJa
...
     if (spanExporterFactory != null) {
       final SpanExporter spanExporter = spanExporterFactory.fromConfig(config);
-      OpenTelemetrySdk.getTracerProvider()
-          .addSpanProcessor(
-              BatchSpansProcessor.create(
-                  spanExporter, BatchSpansProcessor.Config.loadFromDefaultSources()));
+      BatchSpanProcessor spanProcessor =
+          BatchSpanProcessor.newBuilder(spanExporter)
+              .readEnvironmentVariables()
+              .readSystemProperties()
```
Comment (trask) [inline agent-tooling/src/main/java/io/opentelemetry/auto/tooling/TracerInstaller.java:76]: System properties override environment variables 👍

This seems good to me, same as Spring Boot: https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-external-config

Comment (iNikem) [reply agent-tooling/src/main/java/io/opentelemetry/auto/tooling/TracerInstaller.java:76]: Yes :) I always use their order as a reference :)

**Comment by iNikem** [comment]
> LGTM 👍
> 
> Once #463 is done, let's ask @jkwatson to review the whole of `MeterTest`.

Can we at once assign #463 to @jkwatson ? ;)

**Pull request merged**